LOAN/LoanTypes/MortgageLoans.rdf

```diff
@@ -152,57 +519,234 @@
 		<skos:definition xml:lang="en">Designed for senior citizens who want to borrow against the equity of house. The house has been paid for. How is it different?: Scenario based. If you have someone who is asset rich but cash poor then a traditional view of the mortgage wqould say yoiu must have th ecash to repay the loan, which would not apply in this case: we actually don&apos;t anticipate that you would repay it, maybe not even regular amounts on the loan, but when the loan expires, e.g. when you die, we will recover on the asset in the normal course of events. Not seen as a default but is an expectation that we will recover our funds. See Wikipedia. AKA Lifetime morgtage, seniores (age xxx or older) to release the home equity.</skos:definition>
+		<skos:definition>mortgage contract that pays out money to the borrower against a set principal limit that is based on the value of existing equity in the underlying collateral.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>A reverse mortgage and an open end loan both have a credit limit.</fibo-fnd-utl-av:explanatoryNote>
+		<fibo-fnd-utl-av:explanatoryNote>The interest is added to the principal balance.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;RightToBuy">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;MortgageLoanPurpose"/>
-		<rdfs:label xml:lang="en">Right To Buy</rdfs:label>
+	<owl:Class rdf:about="&fibo-loan-typ-mtg;UniversalLoanIdentifier">
+		<rdfs:subClassOf rdf:resource="&fibo-fbc-fi-fi;FinancialInstrumentIdentifier"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-rel-rel;comprises"/>
+				<owl:onClass rdf:resource="&fibo-be-le-lei;LegalEntityIdentifier"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&lcc-lr;identifies"/>
+				<owl:onClass rdf:resource="&fibo-loan-ln-ln;Loan"/>
+				<owl:qualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">1</owl:qualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>universal loan identifier</rdfs:label>
+		<skos:definition>a unique identifier given to unequivocally identify a specific mortgage loan.</skos:definition>
+		<fibo-fnd-utl-av:explanatoryNote>In the US, the structure of this identifier is defined in the 2015 revision to the Home Mortgage Disclosure Act.</fibo-fnd-utl-av:explanatoryNote>
 	</owl:Class>
 	
-	<owl:Class rdf:about="&fibo-loan-typ-mtg;SecondHomeLoan">
-		<rdfs:subClassOf rdf:resource="&fibo-loan-ln-ln;Mortgage"/>
-		<rdfs:label xml:lang="en">second home loan</rdfs:label>
-		<skos:definition xml:lang="en">A loan which may be used for a vacation home or second home used for personal reasons.</skos:definition>
-	</owl:Class>
+	<owl:ObjectProperty rdf:about="&fibo-loan-typ-mtg;assumes">
```

Comment:
unclear name, maybe keep as a synonym. Better is "includesAssumptionOf"

Reply:
Ok - let's raise an issue for that.